admin_ui/src/router.ts

```diff
@@ -8,6 +8,7 @@ import Login from './views/Login.vue'
 import ChangePassword from './views/ChangePassword.vue'
 import RowListing from './views/RowListing.vue'
 import AddForm from './views/AddForm.vue'
+import GetCharts from './views/GetCharts.vue'
```

Comment:
Maybe we just call this view `Chart.vue`?

Reply:
I will change that.